utils/path.js

```diff
@@ -13,11 +13,16 @@ function initStore(initialDirectory = ['blocks']) {
     basePath.pop()
   }
 
+  function resetAllPaths(){
+    basePath.splice(0, basePath.length)
```

Comment:
can this be with slice method.

```
let basePath = [...initialDirectory]
......
basePath = basePath.slice(0, basePath.length)
```

Reply:
No. It is not working as expected if we use the above code.

Reply:
my point is on mutation. nm will have to revisit later on the issue #16 16